pollect/writers/OtelWriter.py

```diff
@@ -0,0 +1,69 @@
+
+from typing import List, Optional
+from pollect.core.ValueSet import ValueSet, Value
+from pollect.sources.Source import Source
+from pollect.writers.Writer import Writer
+
+from opentelemetry.sdk.metrics import MeterProvider
+from opentelemetry.exporter.otlp.proto.http.metric_exporter import OTLPMetricExporter
+from opentelemetry.sdk.metrics.export import PeriodicExportingMetricReader
+from opentelemetry.sdk.metrics import ObservableGauge
+
+
+class OtelWriter(Writer):
+
+    def __init__(self, config):
+        super().__init__(config)
+        self.exporter = OTLPMetricExporter()
+        self.reader = PeriodicExportingMetricReader(self.exporter)
+        self.provider = MeterProvider(metric_readers=[self.reader])
+
+        self.meter = self.provider.get_meter('pollect')
+
+        self.gauges = {}
+
+    def start(self):
+        pass
+
+    def stop(self):
+        pass
+
+    def get_metric_name(self, value_set: ValueSet, value_obj: Value):
+        """
+        Converts the value set and value object to a metric name
+        Replaces all invalid characters with underscores
```

Comment:
Why was this required? Afaik otel allows any utf8 chars as metric names. Do you have a spec for this?

Reply:
It may not necessarily be required, it was basically a take over from `PrometheusWriter`

https://github.com/davidgiga1993/pollect/blob/0f80954c53435427de0b4c9a56685a927df06c94/pollect/writers/PrometheusWriter.py#L181-L186

Will test if its needed

Reply:
okay it can be removed safely

Even when using otel collector + prometheus

```yaml
writer:
  type: Otel
executors:
  - collection: pollect.local
    sources:
      - type: LoadAvg
```

The prometheus itself will do the translation of `pollect.local_loadavg` to `pollect_local_loadavg`

![image](https://github.com/user-attachments/assets/dda1f319-2001-41b0-ae12-105799a7eec9)